### ISSUE TITLE
Accessibility: when use profile picture as drawer icon is turned on talkback doesnt read it

### ISSUE THREAD

**Issue by shortwavesurfer2009** [thunder-app/thunder#1457]
### Bug Description

Normally talkback reads "Open navigation menu" but when "Use profile picture for drawer" is turned on the accessibility label is not applied 

### Expected Behaviour

The accessibility label should be applied to both states where the account photo is used or not used.

### Steps to Reproduce

1. Turn on TalkBack from Settings. Accessibility TalkBack
2. Open Thunder.
3. Open Thunder settings.
4. Go to General.
5. Turn on use profile picture for drawer.
6. Go back.
7. Go to main feed.
8. Tap the Drawer icon, which is now the profile picture.

### App Version

0.5.0-1

### Device

OnePlus Nord N200

### OS

Android13
